source/extensions/http/early_header_mutation/header_mutation/header_mutation.cc

```diff
@@ -25,7 +25,7 @@ class AppendMutation : public Mutation, public Envoy::Router::HeadersToAddEntry
 
     if (!value.empty() || add_if_empty_) {
       switch (append_action_) {
-        PANIC_ON_PROTO_ENUM_SENTINEL_VALUES;
+        PANIC_ON_PROTO_ENUM_SENTINEL_VALUES; // LCOV_EXCL_BR_LINE
```

Comment:
Oooo! Does this actually work? I feel like it used to at one point in the past then stopped.

Reply:
CI says no.  I've tried 2 versions and emailed the bazel team for support though